gen/definitions/profile_parcels/service_routing_ospf.yaml

```diff
@@ -0,0 +1,137 @@
+    test_value: sdwan_service_feature_profile.test.id
+  - model_name: routerId
+    example: 1.2.3.4
+  - model_name: referenceBandwidth
+    example: 101
+  - model_name: rfc1583
+    tf_name: rfc_1583_compatible
+    example: true
+  - model_name: originate
+    example: false
+  - model_name: always
+    example: false
+  - model_name: metric
+    example: 1
+  - model_name: metricType
+    example: type1
+  - model_name: external
+    tf_name: distance_for_external_routes
+    example: 110
+  - model_name: interArea
+    tf_name: distance_for_inter_area_routes
+    example: 110
+  - model_name: intraArea
+    tf_name: distance_for_intra_area_routes
+    example: 110
+  - model_name: delay
+    tf_name: spf_calculation_deplay
+    example: 200
+  - model_name: initialHold
+    tf_name: initial_hold_time
+    example: 1000
+  - model_name: maxHold
+    tf_name: maximum_hold_time
+    example: 10000
+  - model_name: redistribute
+    tf_name: redistributes
+    attributes: 
+    - model_name: protocol
+      id: true
+      example: static
+    - model_name: dia
+      example: true
+    - model_name: refId
+      tf_name: route_policy_id
+      data_path: [routePolicy]
+      exclude_test: true
+      example: 615d948f-34ee-4a2e-810e-a9bd8d3d48ec
+  - model_name: routerLsa
+    tf_name: router_lsas
+    attributes: 
+    - model_name: adType
+      id: true
+      example: on-startup
+    - model_name: time
+      example: 5
+  - model_name: refId
+    tf_name: route_policy_id
+    data_path: [routePolicy]
+    exclude_test: true
+    example: 615d948f-34ee-4a2e-810e-a9bd8d3d48ec
+  - model_name: area
+    tf_name: areas
+    attributes: 
+    - model_name: aNum
+      tf_name: area_number
+      id: true
+      example: 1
+    - model_name: aType
+      tf_name: area_type
+      example: stub
+    - model_name: noSummary
+      example: false
+    - model_name: interface
+      tf_name: interfaces
+      attributes:
+      - model_name: name
+        id: true
+        example: GigabitEthernet2
+      - model_name: helloInterval
+        example: 10
+      - model_name: deadInterval
+        example: 40
+      - model_name: retransmitInterval  
+        tf_name: lsa_retransmit_interval
+        example: 5
+      - model_name: cost
+        tf_name: interface_cost
+        example: 10
+      - model_name: priority
+        tf_name: designated_router_priority
+        example: 1
+      - model_name: network
+        tf_name: ospf_network_type
+        example: broadcast
+      - model_name: passiveInterface
+        example: false
+      - model_name: type
+        example: message-digest
+      - model_name: messageDigestKey
+        tf_name: message_digest_key_id
+        example: 7
+      - model_name: md5
+        tf_name: message_digest_key
+        write_only: true
+        example: sdjfhsghbjdjr
+    - model_name: range
+      tf_name: ranges
+      attributes: 
+      - model_name: ipAddress
+        data_path: [address]
```

Comment:
In OSPFv3 IPv4 we just have "address/mask" here we have "ipAddress/subnetMask" - we should align this in both, not sure which option is better